pymclevel/materials.py

```diff
@@ -944,230 +944,230 @@ def build_indev_materials():
 
 # --- Pocket static block defs ---
 def build_pocket_materials():
-    log.info("Building Pocket materials.")
```

Comment:
Please, pay attention to the indentation character.
We use four spaces for indentation, not tabulation character.
This is important, since Python programs will behave strangely an  may crash if the indentation levels are not strictly respected.

Reply:
okay i'll edit the code and repush.